pom.xml

```diff
@@ -100,6 +100,7 @@ LICENSE file.
     <arangodb.version>2.7.3</arangodb.version>
     <arangodb3.version>4.1.7</arangodb3.version>
     <azurestorage.version>4.0.0</azurestorage.version>
+    <cloudspanner.version>0.9.3-beta</cloudspanner.version>
```

Comment:
No, Cloud Spanner is only released as beta so far. Once we go to a GA release, we will update the version to match the release.